README.md

```diff
@@ -1 +1,15 @@
-# housenet-backend
\ No newline at end of file
+# housenet-backend
+## Getting Started
+> - Download postgresql
+>
+> - Configure the .env file with user and user password
+>
+> - Download relevant packages using pip
```

Comment:
Can just use the requirements.txt which is already committed
pip install -r requirements.txt